packages/workflow/src/Expression.ts

```diff
@@ -51,7 +55,16 @@ export class Expression {
 	 */
 	convertObjectValueToString(value: object): string {
 		const typeName = Array.isArray(value) ? 'Array' : 'Object';
-		return `[${typeName}: ${JSON.stringify(value)}]`;
+
+		if (value instanceof DateTime && value.invalidReason !== null) {
```

Comment:
Can we use [`isDateTime()`](https://moment.github.io/luxon/api-docs/index.html#datetimeisdatetime) here?